cmd/motion/main.go

```diff
@@ -140,9 +140,16 @@ func main() {
 				Name:        "verifiedDeal",
 				Usage:       "whether deals made with motion should be verified deals",
 				DefaultText: "Deals are verified",
-				Value:       true,
+				Value:       false,
 				EnvVars:     []string{"MOTION_VERIFIED_DEAL"},
 			},
+			&cli.BoolFlag{
+				Name:        "keepUnsealed",
+				Usage:       "whether storage providers should retain an unsealed copy for motion deals to make them retrievable",
+				DefaultText: "storage providers WILL maintain an unsealed copy",
+				Value:       true,
+				EnvVars:     []string{"MOTION_KEEP_UNSEALED"},
```

Comment:
i don't think this is something we expose on the CLI like this?

eventually, we get a boost API and have usage support when we ask for unsealing sealed-only content. I don't think we want a global option for the ISV to signal not to do that part? or at least it's not on the requirement control docs i've seen.

if they want it, they should do it by specifying it through their contract with their pool of SPs, not here.

Reply:
yea that's potentially reasonable. part of the reason I exposed it was before we were defaulting to false, which was wrong. No we're defaulting to true -- but I can remove the ability to ever set it false.

Reply:
yeah, lets remove that ability for now.